From 72b7ed65fdf9f2aceca5b7a5dc9b62c3636215d0 Mon Sep 17 00:00:00 2001
From: Teo Hall <teo.hall@nxp.com>
Date: Thu, 26 Jul 2018 15:15:50 -0500
Subject: [PATCH 4252/5242] MLK-19065: arm64: dts: imx8qm: sync operating freq
 with datasheet

commit  2396934f34686803c59c87b4b92a438b8460f475 from
https://source.codeaurora.org/external/imx/linux-imx.git

Sync operating frequencies for A72 as per latest 8QM datasheet.

Signed-off-by: Teo Hall <teo.hall@nxp.com>
(cherry picked from commit 1f9b64ca93014852e42d8e34766f003c361e829b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 432edbc3..ccdebea 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -198,7 +198,6 @@
 		/* kHz    uV */
 		/* voltage is maintained by SCFW, so no need here */
 		1596000	   0
-		1464000	   0
 		1296000	   0
 		1056000	   0
 		600000     0
-- 
1.7.9.5

